pyproject.toml

```
@@ -24,7 +24,7 @@ classifiers = [
 [tool.poetry.dependencies]
 python = ">=3.8,<4.0"
 pydantic = ">=1.9.1"
-fastapi = ">=0.80.0"
+fastapi = "0.93.0"
```

Let's make `fastapi` version not strict:
```suggestion
fastapi = ">=0.93.0"
```